src/third_party/lss/linux_syscall_support.h

```diff
@@ -0,0 +1,4535 @@
+/* Copyright (c) 2005-2011, Google Inc.
```

Comment:
What is the source of this file?

Reply:
The source is from [the official LSS repo](https://chromium.googlesource.com/linux-syscall-support/). The documentation regarding how this ties into Breakpad is located [here](https://chromium.googlesource.com/breakpad/breakpad/+/master/docs/linux_starter_guide.md).

I will update the commit message to make this more clear.

Reply:
Thanks, looks good. Also, you can create a header-only library from it so we can do `hunter_add_package(linux_syscall_support)` instead. Same can be done with `FindDiaSDK`. It can be added to [cmake/find](https://github.com/ruslo/hunter/tree/v0.23.161/cmake/find) with functionality similar to [android_log](https://docs.hunter.sh/en/latest/packages/pkg/android_log.html) or [appkit](https://docs.hunter.sh/en/latest/packages/pkg/appkit.html).